turbo/generators/config.ts

```diff
@@ -72,6 +72,7 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
         validate: validateNotEmpty,
       },
     ],
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
You can remove this disable rule!

Reply:
I couldn't commit until I added this. This is a linting problem, because you used 'any' here:  `actions: (data: any)`. could you please check it?